examples/parallel/parallel.cpp

```diff
@@ -201,6 +203,9 @@ int main(int argc, char ** argv) {
     LOG_TEE("Processing requests ...\n\n");
 
     while (true) {
+        llama_kv_cache_view_update(ctx, &kvc_view);
+        dump_kv_cache_view_seqs(kvc_view, 40);
+
```

Comment:
This stuff needs to go, or at least be controlled by a commandline flag. (I think it would be reasonable to just revert this to the `master` version since trying to add new options is such a massive pain.)

Reply:
> since trying to add new options is such a massive pain.

How come? It's quite easy I think

Reply:
Haha, okay, you got me there. I guess I meant to add them in a way that it's consistent from a user perspective.

The problem is I can do `./main --help` and see the `--dump-kv-cache` option and there's absolutely no way to know it only works for `parallel`. For that reason, I've been very reluctant to try to add any options.

Reply:
BTW, tested the updated version (lightly), seemed okay, so my initial approval still applies.

Reply:
Yeah, that's a small issue, but nothing serious. Btw, we can add this options to `main` - could be useful with loading state from file.